uefi/gberegion.go

```diff
@@ -45,3 +46,14 @@ func (gbe *GBERegion) Extract(parentPath string) error {
 	gbe.ExtractPath, err = ExtractBinary(gbe.buf, dirPath, "gberegion.bin")
 	return err
 }
+
+// Assemble assembles the GBE Region from the binary file.
+func (gbe *GBERegion) Assemble() ([]byte, error) {
+	// We have to trust that the ExtractPath we read from the JSON is correct.
```

Comment:
I'm getting used to this comment everywhere :-)

Reply:
ok I'll remove it. :P